api/types/container/hostconfig.go

```diff
@@ -129,6 +130,17 @@ func (n IpcMode) Container() (idOrName string) {
 	return idOrName
 }
 
+// DefaultNetworkMode returns the default network driver to use for containers
+// on the daemon platform. The default for Linux containers is "bridge"
+// ([network.NetworkBridge]), and "nat" ([network.NetworkNat]) for Windows
+// containers.
+func DefaultNetworkMode() NetworkMode {
```

Comment:
Most uses use `DefaultNetworkMode().NetworkName()` - I'm considering changing this to return the name, instead of `NetworkMode` to simplify most uses.